homeassistant/components/comelit/strings.json

```diff
@@ -31,5 +31,24 @@
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "unknown": "[%key:common::config_flow::error::unknown%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "zone_status": {
```

Comment:
Should we add a name?

Reply:
Zone name are text field that the user can valorise freely; I don0t think there is the possibility to give a name